calendar-be/PlannerApp/src/main/java/com/PlannerApp/PlannerApp/Services/EventService.java

```diff
@@ -80,6 +75,22 @@ public EventDetails getEventDetails(UUID eventId){
                 .build();
     }
 
+
+    public int countGroupEventsOnDate(UUID groupId, java.sql.Date date) {
+        List<EventEntity> groupEvents = eventRepository.getGroupEvents(groupId);
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
+        String targetDateString = formatter.format(date);
+
+        long count = groupEvents.stream()
```

Comment:
why is it long count not int count?

Reply:
to avoid overflow issues

Reply:
But you are returning it still as (int), so will it truly avoid overflow issues?

Reply:
![tarzanas-siksiksik](https://github.com/Lukroza/Planner-App/assets/88896783/cf6d3743-03b9-4e7a-9392-61e166384c85)

Reply:
groupEvents.stream() is long

Reply:
So the function would be more efficient if there were no conversion from long to int and returned long instead